modules/spo-creating-profiles.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-creating-profiles_{context}"]
+= Creating {type} profiles
+
+Use the `{kind}` object to create profiles.
+
+ifdef::seccomp[]
+.Procedure
+
+* The {type} profile will be saved in `/var/lib/kubelet/{type}/operator/<namespace>/<name>.json`.
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: openshift-security-profiles.x-k8s.io/v1beta1
+kind: {kind}
+metadata:
+  namespace: example-namespace
+  name: example-profile
+spec:
+  defaultAction: SCMP_ACT_LOG
+----
+
+An `init` container creates the root directory of the Security Profiles Operator to run the Operator without `root` group or user ID privileges. A symbolic link is created from the rootless profile storage `/var/lib/openshift-security-profiles` to the default `seccomp` root path inside of the kubelet root `/var/lib/kubelet/{type}/operator`.
+endif::[]
+ifdef::selinux[]
+The `{kind}` object has several features that allow for better security hardening and readability. The `{kind}` object:
+
+* Restricts the profiles to inherit from to the current namespace or a system-wide profile. Because there are typically many profiles installed on the system, but only a subset should be used by cluster workloads, the inheritable system profiles are listed in the `spod` instance in `spec.selinuxOptions.allowedSystemProfiles`.
+* Performs basic validation of the permissions, classes and labels.
+* Adds a new keyword `@self` that describes the process using the policy. This allows reusing a policy between workloads and namespaces easily, as the usage of the policy is based on the name and namespace.
+
+.Procedure
+
+. Create a policy that can be used with a non-privileged workload by creating the following `{kind}` object:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: openshift-security-profiles.x-k8s.io/v1alpha2
```

Comment:
`no matches for kind "SelinuxProfile" in version "openshift-security-profiles.x-k8s.io/v1alpha2"`
Above error message will prompt when trying to create the selinuxprofile with the code in this example.

Reply:
Upstream docs refer to several versions: `v1alpha1`, `v1alpha2`, and `v1beta1`. Additionally, I am using the expected downstream name of the api, `openshift-security-profiles`. Upstream is `security-profiles-operator`. 

We will need to update the `apiVersion:` throughout the documentation prior to release. If you know what the version will be, please let me know. Thanks @xiaojiey!